server/conf/i18n/messages

```diff
@@ -466,6 +466,8 @@ dialog.confirmDelete=Are you sure you want to remove this {0}? This change will
 dialog.confirmDeleteAllButtonsSave=Are you sure you want to remove this {0}? This change will be saved when you click "Review", "Previous", or "Next" and all data associated with this {0} will be lost.
 validation.entityNameRequired=Please enter a value for each line.
 validation.duplicateEntityName=Please enter a unique value for each line.
+validation.tooManyEntities=Please enter at most {0} values.
```

Comment:
**suggestion:** Could you add the context strings above each of these. Something like "Validation message to note that the applicant has entered too many / too few values for a question"

Reply:
Thanks - thought of that and then got distracted :| Added!

Reply:
No worries :)